jupyter_server_proxy/config.py

```diff
@@ -152,12 +153,16 @@ class ServerProxy(Configurable):
 
         Value should be a dictionary with the following keys:
           command
-            A list of strings that should be the full command to be executed.
+            An optional list of strings that should be the full command to be executed.
             The optional template arguments {{port}} and {{base_url}} will be substituted with the
             port the process should listen on and the base-url of the notebook.
 
             Could also be a callable. It should return a list.
 
+            If the command is not specified or is an empty list, it is assumed the process is already
+            running, therefore the port checking is skipped and the proxy is set up on the specified
+            port.
```

Comment:
Reflects the change above, and avoids the technical language of "port checking" which also makes the assumption that we don't use a unix socket as could be introduced by #337.

```suggestion
            If the command is not specified or is an empty list, the server
            process is assumed to be started ahead of time and already available
            to be proxied to.
```